noxfile.py

```diff
@@ -0,0 +1,389 @@
+# Copyright iris-grib contributors
+#
+# This file is part of iris-grib and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Perform test automation with nox.
+
+For further details, see https://nox.thea.codes/en/stable/#
+
+"""
+
+from contextlib import contextmanager
+import hashlib
+import os
+from pathlib import Path
+from readline import write_history_file
```

Comment:
@jamesp I don't see where this is used?

Reply:
it's not, looks like an bad auto-import from the text editor.  will remove